src/transformers/models/llava_onevision/processing_llava_onevision.py

```diff
@@ -82,7 +82,7 @@ class LlavaOnevisionProcessor(ProcessorMixin):
         "image_token",
         "video_token",
     ]
-    image_processor_class = "AutoImageProcessor"
+    image_processor_class = "LlavaOnevisionImageProcessor"
```

Comment:
@zucchini-nlp This is because right now, using `AutoImageProcessor.from_pretrained` after saving the models in the kwargs tests will actually load the last preprocessor saved, which is a `LlavaOnevisionVideoProcessor` here. This looks like a larger problem so I'll open a separate issue.

Reply:
Yes, sorry for not noting that when merging the model. Is there any errors caused apart from cases when image and video processors can have different config values?

Reply:
I think we can remove these changes now this has been resolved for the onevision processor